scripts/scylla-get.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+#
+# A platform agnostic Scylla server installation script.
+#
+# To install Scylla, run the following command:
+#
+# curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/scylladb/scylla/master/scripts/scylla-get.sh | sudo bash
+#
+# This file is open source software, licensed to you under the terms
+# of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+# distributed with this work for additional information regarding copyright
+# ownership.  You may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+set -euo pipefail
```

Comment:
`-e` is not effective since you later invoke everything via `main() || exit 1` - the commands will simply continue to be executed because of the `||` operator. Here's an example of a possible weird output:
```bash
[sarna@localhost tmp]$ /tmp/scylla-get.sh --scylla-version 12345
Installing Scylla release 12345 for RHEL ...
Error: This command has to be run with superuser privileges (under the root user on most systems).
Error: This command has to be run with superuser privileges (under the root user on most systems).
Scylla installation done!
```
Also, notice that curl didn't even print that there's no version 12345, because it was invoked silently between the two `yum` invocations that failed due to not being run under `sudo` or as root.